cmd/cainjector/app/start.go

```diff
@@ -55,23 +55,23 @@ func (o *InjectorControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"If set, this limits the scope of cainjector to a single namespace. "+
 		"If set, cainjector will not update resources with certificates outside of the "+
 		"configured namespace.")
-	fs.BoolVar(&o.LeaderElect, "leader-elect", true, ""+
+	fs.BoolVar(&o.LeaderElect, "leader-elect", util.DefaultLeaderElect, ""+
 		"If true, cainjector will perform leader election between instances to ensure no more "+
 		"than one instance of cainjector operates at a time")
-	fs.StringVar(&o.LeaderElectionNamespace, "leader-election-namespace", "", ""+
+	fs.StringVar(&o.LeaderElectionNamespace, "leader-election-namespace", util.DefaultLeaderElectionNamespace, ""+
```

Comment:
This is already explicitly set to `kube-system` in the helm chart (and in the generated manifests).

It may still require a “breaking change” notice for those who don’t use the official manifests/chart, I don’t know.